packages/components/src/form-token-field/index.js

```diff
@@ -583,8 +583,11 @@ class FormTokenField extends Component {
 						/>
 					) }
 				</div>
-				<div id={ `components-form-token-suggestions-howto-${ instanceId }` } className="screen-reader-text">
-					{ __( 'Separate with commas' ) }
+				<div id={ `components-form-token-suggestions-howto-${ instanceId }` } className="components-form-token-field__help">
```

Comment:
Since we're touching this part, I'd change this `<div>` to a `<p>`: slightly better from a semantic perspective.